mlflow/server/js/src/components/CompareRunView.js

```diff
@@ -153,7 +156,21 @@ class CompareRunView extends Component {
     });
 
     return keys.map(k => {
-      return <tr key={k}>
+      let row_class = undefined;
+      if (highlightChanges) {
+        const all_equal = data[k].reduce((acc, currentValue) => {
```

Comment:
```suggestion
        data[k].every(x => x == data[k][0])
```

I think the reduce is unnecessarily confusing. Could you just compare each value to the 0'th value?

Reply:
Good point. I simplified the reduce